packages/components/src/tooltip/test/index.js

```diff
@@ -97,37 +97,31 @@ describe( 'Tooltip', () => {
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-
-			let event;
-
-			event = { type: 'mousedown' };
-			button.simulate( event.type, event );
+			const button = screen.getByRole( 'button' );
+			fireEvent.mouseDown( button );
 			expect( originalOnMouseDown ).toHaveBeenCalledWith(
 				expect.objectContaining( {
-					type: event.type,
+					type: 'mousedown',
 				} )
 			);
 
-			event = { type: 'focus', currentTarget: {} };
-			button.simulate( event.type, event );
-
-			const popover = wrapper.find( 'Popover' );
+			button.focus();
+			const popover =
+				container.getElementsByClassName( 'components-popover' );
```

Comment:
I used `getElementsByClassName` bedause there were no accessible attributes for the popover (then eventCatcher),.
Are there any other recommended selectors?

Reply:
If we mock the `Popover` component, we can add a `data-testid` in the mock there, and be able to query by `getByTestId` or `queryByTestId`. That should help replace most of those `container.getElementsByClassName` in the rest of this PR.

Reply:
Let's try to avoid mocking the popover component — instead, we can check that the Tooltip text is displayed or not

Reply:
Fixed by d6f98a7👍